google-cloud-bigquery/lib/google/cloud/bigquery/copy_job.rb

```diff
@@ -129,6 +129,105 @@ def write_empty?
           disp = @gapi.configuration.copy.write_disposition
           disp == "WRITE_EMPTY"
         end
+
+        ##
+        # Yielded to a block to accumulate changes for an API request.
+        class Updater < CopyJob
+          ##
+          # @private Create an Updater object.
+          def initialize gapi
+            @gapi = gapi
+          end
+
+          ##
+          # @private Create an Updater from an options hash.
+          #
+          # @return [Google::Cloud::Bigquery::CopyJob::Updater] A job
+          #   configuration object for setting copy options.
+          def self.from_options source, target, options = {}
+            req = Google::Apis::BigqueryV2::Job.new(
+              configuration: Google::Apis::BigqueryV2::JobConfiguration.new(
+                copy: Google::Apis::BigqueryV2::JobConfigurationTableCopy.new(
+                  source_table: source,
+                  destination_table: target
+                ),
+                dry_run: options[:dryrun]
+              )
+            )
+
+            updater = CopyJob::Updater.new req
+            updater.create = options[:create]
```

Comment:
Does this need to be converted with ` Convert.create_disposition(options[:create])`?

(Is there an existing test covering these assignments?)

Reply:
Yes it needs to be converted, but this conversion is done in `def create=`.

Yes, it appears this behavior is tested at https://github.com/GoogleCloudPlatform/google-cloud-ruby/blob/f479fbcd3a280df07ec85218a7c98f76c5d40697/google-cloud-bigquery/test/google/cloud/bigquery/table_copy_test.rb#L123 and https://github.com/GoogleCloudPlatform/google-cloud-ruby/blob/f479fbcd3a280df07ec85218a7c98f76c5d40697/google-cloud-bigquery/test/google/cloud/bigquery/table_copy_test.rb#L156

Reply:
Oh yes of course. I saw that, but then mistakenly thought this assignment was to the gapi object. Thanks.